sql/account_ecosystem/aet_clients_daily/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+friendly_name: AET Clients Daily
+description: >
+  One row per user per service per day, showing metrics across services.
+
+  The `user_id` and `client_id` are directly related to the `ecosystem_user_id`
```

Comment:
Note the use of simple `user_id` and `client_id` as the field names for the final user-facing analysis table. I don't want to call them `ecosystem_user_id` and `ecosystem_client_id` because they are not directly those values. `canonical_id` seems jargony and the details of how we maintain continuity across password resets are not necessary for end users to think about.

I am very open to feedback on this naming decision, and also on the naming for `service` which ends up being "fxa - <some_oauth_client_id>" or "desktop". Calling desktop a "service" doesn't seem right. Maybe "source", but that seems _too_ generic.